projects/fsastorefrontlib/src/cms-components/agent/agent-search-list/agent-search-list.component.ts

```diff
@@ -57,10 +65,19 @@ export class AgentSearchListComponent implements OnInit, OnDestroy {
   }
 
   setActiveAgentIndex(selectedIndex: number) {
+    if (selectedIndex === -1) {
+      this.selectedAgentState = false;
+      this.getAgents(this.searchQuery);
+    }
     this.selectedIndex = selectedIndex;
   }
 
+  getAgents(searchValue: string) {
+    this.agentSearchService.search(searchValue, 0);
+  }
+
```

Comment:
Seems this method is not needed as it just passes the value to another method in this component?